includes/resourceloader/ResourceLoader.php

```diff
@@ -715,9 +715,14 @@ protected function makeComment( $text ) {
 	 * @param Exception $e to be shown to the user
 	 * @return string sanitized text that can be returned to the user
 	 */
-	protected function formatException( $e ) {
+	protected function formatException( Exception $e ): string {
 		global $wgShowExceptionDetails;
 
+		// SUS-1899: Ensure we log exceptions thrown in ResourceLoader
+		\Wikia\Logger\WikiaLogger::instance()->error( $e->getMessage(), [
```

Comment:
Let's log `__METHOD__` instead of exception message to avoid polluting Jira with auto-generated tickets from error log.